Sources/Beacon/Options.cs

```diff
@@ -35,6 +35,12 @@ internal class Options
         [Option("timespan", Default = 7, HelpText = "The timespan in days to include builds from.")]
         public int Timespan { get; set; }
 
+        [Option('a', "active", Default = false, HelpText = "Monitor builds for active branches only.")]
+        public bool ActiveBranchesOnly { get; set; }
+        
+        [Option('f', Default = false, HelpText = "Include failed to start builds.")]
```

Comment:
🔧 Maybe the help text should be a bit more clear what _failed to start_ means. Now it just repeats the name of the option.